src/View/Game/_gamecard.html.twig

```diff
@@ -29,7 +29,10 @@
 			</p>
 		</div>
 		<div class="card-footer text-center w-100 d-flex justify-content-around">
-			{% if game.availability == true %}
+			{% if game.availability == true and game.game_id in requestedGames == true %}
```

Comment:
the `== true` is not necessary

Reply:
I changed it. But i think it's more reader-friendly to write it, no ?